exporter/awskinesisexporter/internal/batch/aggregator.go

```diff
@@ -0,0 +1,75 @@
+package batch
+
+import (
+	"crypto/md5"
+
+	k "github.com/aws/aws-sdk-go/service/kinesis"
+	"google.golang.org/protobuf/proto"
+)
+
+var (
+	magicNumber           = []byte{0xF3, 0x89, 0x9A, 0xC2}
+	partitionKeyIndexSize = 8
+	maxAggregationCount   = 4294967295
+	maxRecordSize         = 1 << 20
+)
+
+type Aggregator struct {
+	buf    []*Record
+	pkeys  []string
+	nbytes int
+}
+
+func (a *Aggregator) IsRecordAggregative(data []byte, partitionKey string) bool {
```

Comment:
This function name throws me off a bit, I don't really understand what the method names or what situation to use it in.

Could I ask you to comment the exported methods here?